mantidproject/mantid
# Transverse field mode in Muon Analysis

**Description of work.**
The TF asymmetry mode in muon analysis was requested to explicitly show the fitting function being used. These changes are to allow the full normalization function to be shown. 

**Report to:** Peter. 

**To test:**


Test 1

- Open Muon analysis
- Load MUSR62260
- In settings make sure multiple fit is unticked
- Go to data analysis
- Add a GausOsc function and set the frequency to 1.3
- Do a fit (this should work as normal).
- Click TF Asymmetry mode in the settings table
- the function will have changed and the plot updated
- The GausOsc should be in the new function and have the same values as before
- Run fit
- The first flat background value will have changed and should match the normalization in MuonAnalysisTFNormalizations table
- Turning TF Asymmetry mode off will result in the function changing back to just a GausOsc (with updated values).
- In results table tab the _unNorm workspace should be available as a fitting result
 
Test 2
- Open Muon Analysis
- Load MUSR62260
- In settings make sure multiple fit is unticked
- Go to data analysis
- Add a GausOsc function and set the frequency to 1.3 
- Set frequency and sigma to global
- Click TF Asymmetry mode in the settings table
- The new function should include the GausOsc function and have preserved the global ticks
- Run a fit
- The first flat background values (in each of the groups)  will have changed and should match the normalization in MuonAnalysisTFNormalizations table
- Turning TF Asymmetry mode off will result in the function changing back to just a GausOsc (with updated values).
- In results table tab click multiple (middle of page)
- The output should be available for creating a table

Fixes #21718 . 

Does this update require release notes?
- [x ] Yes

In release notes and doc updated.
---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
